pkg/controller/cluster/controller.go

```diff
@@ -406,6 +427,18 @@ func isEqualExecProviderConfig(p *v1alpha1.ExecProviderConfig, r *argocdv1alpha1
 	return true
 }
 
+func isEqualFromKubeconfig(p *v1alpha1.ClusterParameters, r *argocdv1alpha1.Cluster) bool {
```

Comment:
This function does not seem to be used anywhere.

Reply:
Removed